sdk/state/open.go

```diff
@@ -49,32 +49,7 @@ type OpenParams struct {
 	ExpiresAt                  time.Time
 }
 
-func (c *Channel) OpenTxs(d OpenAgreementDetails) (txClose, txDecl, formation *txnbuild.Transaction, err error) {
-	txClose, err = txbuild.Close(txbuild.CloseParams{
-		ObservationPeriodTime:      d.ObservationPeriodTime,
-		ObservationPeriodLedgerGap: d.ObservationPeriodLedgerGap,
-		InitiatorSigner:            c.initiatorSigner(),
-		ResponderSigner:            c.responderSigner(),
-		InitiatorEscrow:            c.initiatorEscrowAccount().Address,
-		ResponderEscrow:            c.responderEscrowAccount().Address,
-		StartSequence:              c.startingSequence,
-		IterationNumber:            1,
-		AmountToInitiator:          0,
-		AmountToResponder:          0,
-		Asset:                      d.Asset.Asset(),
-	})
-	if err != nil {
-		return
-	}
-	txDecl, err = txbuild.Declaration(txbuild.DeclarationParams{
-		InitiatorEscrow:         c.initiatorEscrowAccount().Address,
-		StartSequence:           c.startingSequence,
-		IterationNumber:         1,
-		IterationNumberExecuted: 0,
-	})
-	if err != nil {
-		return
-	}
+func (c *Channel) openTxs(d OpenAgreementDetails) (formation *txnbuild.Transaction, err error) {
```

Comment:
I want to change the name of the formation tx to the open tx 😄, but I thought I'd keep that for a separate PR because I thought that might cause too many conflicts at this point.

Reply:
The why behind wanting to rename it is from the user's perspective there is only one open transaction, which right now is called the formation transaction. I can see conversations like this "what's the formation tx" "oh its the open tx". Seems like we can shortcut that indirection.

I think it is okay that the "open process" will involve signing the "close txs" and the "open tx".

Reply:
ahh gotchya ok. In that case makes sense 😄 

could we change it to `openTx` instead of `openTxs` though? 😄 

Going to approve now though